sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_azure_appconfiguration_client.py

```diff
@@ -233,12 +225,8 @@ def get_configuration_setting(
         :type match_condition: ~azure.core.MatchConditions
         :keyword str accept_datetime: retrieve ConfigurationSetting existed at this datetime
         :return: The matched ConfigurationSetting object
-        :rtype: ~azure.appconfiguration.ConfigurationSetting
-        :raises: :class:`~azure.core.exceptions.HttpResponseError`, \
-            :class:`~azure.core.exceptions.ClientAuthenticationError`, \
-            :class:`~azure.core.exceptions.ResourceNotFoundError`, \
-            :class:`~azure.core.exceptions.ResourceModifiedError`, \
-            :class:`~azure.core.exceptions.ResourceExistsError`
+        :rtype: ~azure.appconfiguration.ConfigurationSetting or None
```

Comment:
They are still possible errors.

We refactored the code but did not change the behavior.

Reply:
I know. I kept the `HttpResponseError` which is the parent error type, it aligned with the rtype in generated code docstring. I'm not sure if we need to specify every error type here.

Reply:
I don't think we should change the doc and then tell customers nothing changed. :)

Reply:
Make sense, will revert them then.

Reply:
> They are still possible errors.
 
I don't think it's error, the docstring was incorrect, it does not align with the type hint in this method. The return value could be `None` when `ResourceNotModifiedError` occured.